tedge/src/cli/certificate/cli.rs

```diff
@@ -81,5 +85,9 @@ pub enum UploadCertCli {
         /// Provided username should be a Cumulocity user with tenant management permissions.
         /// The password is requested on /dev/tty, unless the $C8YPASS env var is set to the user password.
         username: String,
+
+        #[structopt(long = "no-verify-server")]
+        /// Allow upload to a server with invalid certificate, default = false.
+        ignore_server_certficate_validity: bool,
```

Comment:
I'm not fully sure if we need this. In the Thick Edge case described in the story, if they add their certificate to the local of a Thin Edge device, can't it be solved?

Reply:
They generate invalid certificate from the point of view our certificate verifier and adding root to trusted doesn't resolve the issue. For the time being ignoring invalid server certificate will.

Reply:
I think @rina23q is right. It's not clear why we are adding this extra option to allow bad certificates. Just because one thick-edge use case is broken right now is not a reason.